packages/yourloops/pages/profile/personal-info-form.tsx

```diff
@@ -25,136 +25,91 @@
-        {props.user.frProId && <CertifiedProfessionalIcon id={`certified-professional-icon-${props.user.id}`} />}
+        <strong className={classes.uppercase}>{t('personal-information')}</strong>
+        {user.frProId && <CertifiedProfessionalIcon id={`certified-professional-icon-${user.id}`} />}
       </Box>
 
-      <Box className={props.classes.inputContainer}>
+      <Box className={classes.inputContainer}>
         <TextField
           id="profile-textfield-firstname"
           label={t('firstname')}
-          value={props.firstName}
-          onChange={event => props.setFirstName(event.target.value)}
-          error={props.errors.firstName}
-          helperText={props.errors.firstName && t('required-field')}
-          className={`${props.classes.formInput} ${props.classes.halfWide}`}
+          value={profileForm.firstName}
+          onChange={event => updateProfileForm(ProfileFormKey.firstName, event.target.value)}
+          error={errors.firstName}
+          helperText={errors.firstName && t('required-field')}
+          className={classes.formInput}
         />
         <TextField
           id="profile-textfield-lastname"
           label={t('lastname')}
-          value={props.lastName}
-          onChange={event => props.setLastName(event.target.value)}
-          error={props.errors.lastName}
-          helperText={props.errors.lastName && t('required-field')}
-          className={`${props.classes.formInput} ${props.classes.halfWide}`}
+          value={profileForm.lastName}
+          onChange={event => updateProfileForm(ProfileFormKey.lastName, event.target.value)}
+          error={errors.lastName}
+          helperText={errors.lastName && t('required-field')}
+          className={classes.formInput}
         />
       </Box>
 
-      {props.role === UserRoles.hcp &&
-        <Box className={props.classes.inputContainer}>
-          <Box className={`${props.classes.formInput} ${props.classes.halfWide}`}>
+      {user.isUserHcp() &&
+        <Box className={classes.inputContainer}>
+          <Box className={classes.formInput}>
             <BasicDropdownWithValidation
-              onSelect={props.setHcpProfession}
-              defaultValue={props.hcpProfession}
+              onSelect={(value: string) => updateProfileForm(ProfileFormKey.hcpProfession, value)}
+              defaultValue={profileForm.hcpProfession}
               disabledValues={[HcpProfession.empty]}
               values={HcpProfessionList.filter(item => item !== HcpProfession.empty)}
               id="profession"
+              dataTestId="hcp-profession-selector"
               inputTranslationKey="hcp-profession"
               errorTranslationKey="profession-dialog-title"
             />
           </Box>
 
-          {appConfig.ECPS_ENABLED && props.user.settings?.country === 'FR' &&
+          {appConfig.ECPS_ENABLED && user.settings?.country === 'FR' &&
```

Comment:
Same comment about the raw string `'FR'` used here as in `patient-form.tsx`, the code would really be more robust if this string was defined in a single place. Now that I see that it needs to be used in multiple files, the `enum` solution looks like the best one 😄